tests/robot-tests/tests/libs/admin_api.py

```diff
@@ -206,6 +206,13 @@ def user_creates_test_release_via_api(
     return response.json()["id"]
 
 
+def user_updates_release_published_date(release_id: str, published: datetime) -> None:
```

Comment:
I think we should add a `_via_api` on to this. I think it is worth making it clear what is done through selenium and what is not.